src/components/OcTextarea.vue

```diff
@@ -1,51 +1,145 @@
 <template>
-  <textarea
-    class="uk-textarea"
-    :value="value"
-    :placeholder="placeholder"
-    :aria-label="label"
-    @input="$_ocTextArea_onInput($event.target.value)"
-    @focus="$_ocTextArea_onFocus($event.target.value)"
-    @keydown="$_ocTextArea_onKeyDown($event)"
-  />
+  <div>
+    <label class="oc-label" :for="textareaId" v-text="label" />
+    <textarea
+      :id="textareaId"
+      v-bind="additionalAttributes"
+      class="oc-textarea"
+      :class="{
+        'oc-textarea-warning': !!warningMessage,
+        'oc-textarea-danger': !!errorMessage,
+      }"
+      :value="value"
+      :aria-invalid="ariaInvalid"
+      @input="onInput($event.target.value)"
+      @focus="onFocus($event.target.value)"
+      @keydown="onKeyDown($event)"
+    />
+    <div v-if="showMessageLine" class="oc-textarea-message">
+      <span
+        :id="messageId"
+        :class="{
+          'oc-textarea-description': !!descriptionMessage,
+          'oc-textarea-warning': !!warningMessage,
+          'oc-textarea-danger': !!errorMessage,
+        }"
+        v-text="messageText"
+      ></span>
+    </div>
+  </div>
 </template>
 
 <script>
+import uniqueId from "../utils/uniqueId"
+
 /**
  * Textareas are used to allow users to provide text input when the expected
  * input is long. Textarea has a range of options. For shorter input,
  * use the `Input` element.
  *
  * ## Accessibility
- * The attributes `placeholder` and `aria-label` have different functions. The first specifies a short hint describing the expected value of an input field/text area, or gives an example (e.g. email@example.com). `aria-label` provides the accessible name of the textarea (e.g. "Your address", "Comment",...).
+ * The label is required and represents the name of the textarea.
+ *
+ * The description-message can be used additionally to give further information about the textarea . When a
+ * description is given, it will be automatically referenced via the `aria-describedby` property.
+ * An error or warning will replace the description as well as the `aria-describedby` property until the error
+ * or warning is fixed.
  */
 export default {
   name: "OcTextarea",
   status: "review",
   release: "1.0.0",
   props: {
     /**
-     * Text value of the form textarea.
+     * The ID of the element.
      */
-    value: {
+    id: {
```

Comment:
We have already an id here, replace the default with unique-id and rename old textareaId :+1: